stdlib.ispc

```diff
@@ -711,25 +711,31 @@ __declspec(safe) static inline int64 count_trailing_zeros(int64 v) {
 // AOS/SOA conversion
 
 static inline void aos_to_soa2(uniform float a[], varying float *uniform v0, varying float *uniform v1) {
-    __aos_to_soa2_float(a, v0, v1);
```

Comment:
What about defining `opaque_ptr_t` and using it though the file? Otherwise it's too ugly.

Reply:
Good idea, thanks!

Reply:
The commit with `opaque_ptr_t` looks good.